050-Manage/030-migrations.mdx

```diff
@@ -73,10 +73,131 @@ Migration files should not be modified. If externally modified, the record will
+
+<Alert status="warning">
+  Currently multi-version schema migrations are only supported via the Migration Editor in the UI. Migrations made via
+  the Table View and Schema View will complete automatically.
+</Alert>
+
+![The Migration Editor](images/migrations-multiversion/migration-editor.png)
+
+Some common schema operations are:
+
+<TabbedCode tabs={['create_table', 'add_column', 'rename_table', 'alter_column', 'drop_table', 'drop_column']}>
+```jsonc
+[{
+  "create_table": {
+    "name": "customers",
+    "columns": [
+      {
+        "name": "id",
+        "type": "integer",
+        "pk": true
+      },
+    ]
+  }
+}]
+```
+```jsonc
+[{
+  "add_column": {
+    "table": "customers",
+    "up": "UPPER(name)",
+    "column": {
+      "name": "full_name",
+      "type": "varchar(255)",
+      "nullable": true
+    }
+  }
+}]
+```
+```jsonc
+[{
+  "rename_table": {
+    "from": "customers",
+    "to": "clients"
+  }
+}]
+```
+```jsonc
+[{
+  "alter_column": {
+    "table": "clients",
+    "column": "full_name",
+    "name": "new_full_name",
+    "up": "(SELECT CASE WHEN full_name IS NULL OR LENGTH(full_name) <= 3 THEN 'placeholder' ELSE full_name END)",
+    "down": "full_name"
+  }
+}]
+```
+```jsonc
+[{
+  "drop_table": {
+    "name": "clients"
+  }
+}]
+```
+```jsonc
+[{
+  "drop_column": {
+    "table": "clients",
+    "column": "full_name",
+    "down": "''"
+  }
+}]
+```
+</TabbedCode>
 
 See the [pgroll Operations Reference](https://github.com/xataio/pgroll/blob/main/docs/README.md#operations-reference) for the list of supported migration operations.
+
+## Lifecycle
+
+A multi-version schema migration's life cycle consists of `start`, `complete`, and `rollback` steps.
+
+### Start Step
+
+After a migration is started it is considered `active`. One active migration per branch is possible. This is reflected in disabled buttons and inputs in the UI while a migration is ongoing. From there you can decide whether to `complete` or `rollback` the active migration.
+
+You can view the submitted migration under the Schema History tab while it is `active`.
+
+![An active migration under the Schema History tab](images/migrations-multiversion/active-migration.png)
+
+As more data is added to the table through either the old or the new version of the schema, the ‘other’ column is automatically updated by `pgroll` by the `up` or `down` triggers. Data in each version of the column remains editable.
```

Comment:
What's the other column in this case. Right now at least, the screenshot below does not have an other column and is called `newcol`. I'm not sure if this is meant ot reference it or simply referring to a new column generally

Reply:
Clarified